dna/models/baselines.py

```diff
@@ -249,9 +249,15 @@ def _process_runs(data):
 
     @staticmethod
     def _process_metafeatures(data):
```

Comment:
The changes in this function were necessary in order to make the `autosklearn` metamodel approach work for the D3M data. It used to not consider the `dataset_id` when identifying duplicates, and would end up dropping some datasets and their metafeatures when operating on the D3M data. This is because in the D3M data, some datasets have identical metafeatures. I haven't looked into which ones.

Reply:
It is very improbable for metafeatures to be the same unless the data is also the same. Unless maybe we have very simplistic metafeatures for more exotic data types (not numeric or categorical). Can you look into that quickly?

Reply:
I assumed the culprits were the same datasets having different `dataset_id` fields. It will be good to know for sure. I'll look into it.

Reply:
It looks like that is the case. There are some duplicate datasets under different `dataset_id`s. Here are the duplicate groups the current D3M DB data I'm using has:

LL0_1442_MegaWatt1_dataset
LL0_1449_MeanWhile1_dataset

LL0_40478_thyroid_dis_dataset
LL0_40474_thyroid_allbp_dataset
LL0_40475_thyroid_allhyper_dataset

LL0_1466_cardiotocography_dataset
LL0_1560_cardiotocography_dataset

LL0_1523_vertebra_column_dataset
LL0_1524_vertebra_column_dataset

LL0_1499_seeds_dataset
LL0_1500_seismic_bumps_dataset

Reply:
Thank you Evan! I will forward this to d3m and see what can be done to clean this up. On our end, is there anything else we should do?